src/structures/game_round.ts

```diff
@@ -530,14 +532,15 @@ export default class GameRound extends Round {
                     ? ` (+${friendlyFormattedNumber(playerResult.expGain)} EXP)`
                     : "";
 
+                const timeToGuess = durationSeconds(this.startedAt, createdAt);
```

Comment:
Don't we already calculate guess times somewhere else? We should re-use so that there's only one source of truth

Reply:
`createdAt` is the most recent guess we have, from above. A similar variable `earliestGuess` exists for the non-hidden code path, and that is used below